package.json

```diff
@@ -39,6 +39,7 @@
     "enzyme": "2.2.0",
     "eslint": "2.4.0",
     "eslint-config-airbnb": "6.2.0",
+    "eslint-plugin-import": "^1.4.0",
```

Comment:
I think we should use `"1.4.0"` to avoid different people using different versions of the module.